Decred Wallet/Features/Wallets/WalletsViewController.swift

```diff
@@ -22,22 +22,27 @@ class WalletsViewController: UIViewController {
         self.walletsTableView.registerCellNib(WalletInfoTableViewCell.self)
         self.walletsTableView.dataSource = self
         self.walletsTableView.delegate = self
+        NotificationCenter.default.addObserver(self,
```

Comment:
don't you think this is better to be added only when the backup wallet wallet button is clicked and then removed after the operation is done.

Reply:
I had not seen the delegate methods of the wallets info cell. I have refactored. Should be good now.